FitGoal/FitGoal/UserProfileConfiguration/WalkthroughIcon.swift

```diff
@@ -50,5 +52,6 @@ struct WalkthroughIcon {
 
 enum WalkThroughIconType {
     case gender(Gender)
-    case fitnessLevel(FitnessLevel)
+    case currentFitnessLevel(FitnessLevel)
+    case fitnessGoal(FitnessLevel)
```

Comment:
I think these two names aren't consistent. In the first one you use the type name as a suffix but in the second you only use a part of the type name. 
If _fitness_ has a meaning in itself for you (witch seems to be the case), then you could use `currentFitness` and `fitnessGoal`. If it has not meaning, then you should use `currentFitnessLevel` and `fitnessLevelGoal`. 
I prefer the first option, but the decision is up to you.